android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/webextension/WebExtensionDelegate.kt

```diff
@@ -135,4 +135,10 @@ interface WebExtensionDelegate {
      * of installing / uninstalling extensions by the user.
      */
     fun onExtensionListUpdated() = Unit
+
+    /**
+     * Invoked when the extension process spawning has been blocked. This can occur because
```

Comment:
```suggestion
     * Invoked when the extension process spawning has been disabled. This can occur because
```

but I think a rebase will fix this nit